servicex/servicex.py

```diff
@@ -37,7 +37,7 @@ class ServiceXDataset(ServiceXABC):
     '''
     def __init__(self,
                  dataset: str,
-                 image: str = 'sslhep/servicex_func_adl_xaod_transformer:130_reset_cwd',
+                 image: str = 'sslhep/servicex_func_adl_xaod_transformer:29_dont_truncate_error_logs',  # NOQA
```

Comment:
Should this be 1.0.0-RC.2 for now?

Reply:
This will get fixed as soon as there is a working tag of that name! So not in time for this PR, but shortly after that, I hope!